metrics/metrics-common/src/main/java/org/apache/servicecomb/metrics/common/MetricsPublisher.java

```diff
@@ -17,43 +17,8 @@
 
 package org.apache.servicecomb.metrics.common;
 
-import java.util.List;
+import java.util.Map;
```

Comment:
common move to foundation-metrics

Reply:
Done

Reply:
metrics-common had deleted and move all content into foundation-metrics